### PR TITLE
Give more hints on setting up Amazon SNS topic subscriptions

### PR DESCRIPTION
**Pull request by aspiers**
I spent a couple of hours scratching my head why I couldn't get Amazon to confirm my topic subscription.  So let's try to avoid others having the same struggle.

### CODE REVIEW

File: en/emails/bounce_management.md [hunk 1]
```diff
@@ -70,7 +70,40 @@ Mautic supports the bounce and complaint management from Amazon Simple Email Ser
 
 ![Topic](/emails/media/amazon_webhook_5.png "Enter url to Mautic")
 
-4) The subscriber will be in the pending state till it is confirmed. AWS will call your Amazon webhook with a SubscriptionConfirmation request including a callback url. To confirm Mautic will send a request back to this callback url to validate the subscription. Therefore make sure your Mautic installation is allowed to connect to the internet, otherwise the subscription will remain in the pending state and won't work. Check the logfile for more information.
+4) The subscriber will be in the pending state till it is
```
Comment (escopecz) [inline en/emails/bounce_management.md]: Why the new lines?

Comment (aspiers) [reply en/emails/bounce_management.md]: -   It makes future diffs much easier to read.
-   It avoids words being split in half due to line wrapping.
-   It ensures [optimal readability](https://baymard.com/blog/line-length-readability) even if the editor or viewer is wider.
-   It ensures all the text is in view, even if the viewer or editor is configured not to line wrap.
-   (Slightly weaker point, but still has some truth to it:) Based on many years of working on FOSS projects (especially on GitHub), it's the more common practice I've seen adopted by coders.  Even though it originally applied to code, e.g.

    - https://www.python.org/dev/peps/pep-0008/#maximum-line-length
    - https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings 
    - http://katafrakt.me/2017/09/16/80-characters-line-length-limit/

    through natural desire for consistency within a repository, it's usually applied to the docs too.

-   (Again a weaker point but still somewhat valid:) On some editors, the "move to next line" command will move to the next *logical* line, not the next physical line (i.e. the next one visible on the screen when line-wrapping is enabled), so jumping quickly from (say) near the beginning of a paragraph to near the end can't be done via up/down motion unless the paragraph is line-wrapped.

The first three points are the strongest for me.  The others are less significant but still lend weight to the argument.

Having said all that, if line-wrapping is a deal-breaker for you, I can revert it.  The main thing is to get the extra info in :-)

Comment (escopecz) [reply en/emails/bounce_management.md]: Interesting points. My only point against was consistency with the rest of the docs. But I'm fine with it. Thanks!

Comment (aspiers) [reply en/emails/bounce_management.md]: @escopecz Indeed, I agree that consistency is important.  If you like I could line-wrap other content to match?

Comment (escopecz) [reply en/emails/bounce_management.md]: I don't think it's necessary

**Pull request merged**